src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/global_navitagtion/NotificationsComponentObject.java

```diff
@@ -1,8 +1,7 @@
 package com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00120)